nvflare/tool/job/config/configer.py

```diff
@@ -312,6 +324,32 @@ def _parse_cli_config(cli_configs: List[str], app_names: List[str]) -> Dict[str,
     return app_cli_config_dict
 
 
+def get_config_file_path(app_name, input_file_path, job_folder):
+    basename = os.path.basename(input_file_path)
+    if basename.startswith(f"{JOB_META_BASE_NAME}."):
+        config_file = os.path.abspath(os.path.join(job_folder, basename))
+    else:
+        # The input_file_path could be in one of the following format
```

Comment:
what will happen if users provide an input_file_path starts from the root directory "/hello/cool.conf"?